rpc/dcrwallet.go

```diff
@@ -121,6 +121,13 @@ func (w *WalletConnect) Clients(ctx context.Context, netParams *chaincfg.Params)
 			continue
 		}
 
+		if !walletInfo.ManualTickets {
+			// All wallet's should not be adding tickets found via the network.
+			log.Errorf("wallet does not have manual tickets enabled (wallet=%s)", c.String())
+			failedConnections = append(failedConnections, connect.addr)
```

Comment:
I don't think this needs to be counted as a failed wallet connection. All of the wallet RPCs will still be available, and as we discussed on Matrix, misconfiguration here will not have any negative impact on VSP users, only (potentially) the service operator.

Should be fine to just log an error and proceed with using the connection - see below for `.Voting` and `.Unlocked` examples.